src/duties/browserResults.js

```diff
@@ -0,0 +1,48 @@
+let _ = require('lodash/fp')
+
+let hasBrowserErrors = ({ browserResults, fail, message }) => {
+  if (_.isNil(browserResults)) {
+    message('Could not find any browser results.')
+    return
+  }
+
+  let { result, summary } = browserResults
+
+  if (result && summary) {
+    let { error, failed, exitCode } = summary
+    if (error === false && failed === 0 && exitCode === 0) {
+      message('Your PR has no browser errors. Great job!')
+      return
+    }
+
+    if (failed > 0) {
+      _.flow(
+        // Flatten test results into one array.
+        _.reduce((flattened, other) => {
+          return flattened.concat(other)
```

Comment:
just return directly after the arrow function. You should probably also concat using spreads eg:

`(flattened, other) => [...flattened, other]`

Reply:
Tried this `_.reduce((flattened, other) => [...flattened, other], []),` but doesn't work.  I should probably change the spec to match the output text.

Reply:
Btw, what you want me to do with the package.json?

Reply:
`"lint": "eslint . --ignore-path .gitignore",` ?

Reply:
ah, that's fine then. `_.reduce((flattened, other) => flattened.concat(other), [])` should do the trick. Just trying to avoid the unnecessary `return` there 👍

Reply:
also, in package.json, I inlined the options `--singleQuote` etc in the fmt script, but since we're breaking configuration out, we should remove it in the `fmt` script

Reply:
`"fmt": "prettier --write {*.js,{src,bin}/**/*.js} --no-semi --single-quote --trailing-comma es5"`

turns to

`"fmt": "prettier --write {*.js,{src,bin}/**/*.js}"`

Reply:
That looks way better.